docs/interpreter_status.md

```diff
@@ -0,0 +1,216 @@
+# StableHLO Interpreter
+
+The main goal of the StableHLO interpreter is to provide a reference
+implementation to the semantics of StableHLO opset according to its
+specification. The secondary goal is for the implementation to closely follow
+the spec to provide additional clarity to the semantics of even the most
+involved operations like `Convolution`, `Gather`/`Scatter`, and `DotGeneral`.
+
+At the moment, OpenXLA supports the interpretation of 91 out of 96 specced
+StableHLO ops, with the remaining 5 ops (`FftOp`, `RngOp`, `RngBitGeneratorOp`,
+`UniformDequantizeOp`, `UniformQuantizeOp`) being a work in progress (see
```

Comment:
is "work in progress" accurate?

Perhaps "initial investigation completed and documented in <link tickets>, these final enhancements will be implemented on an as-needed community basis"